Source/ACE.Server/WorldObjects/Player_Inventory.cs

```diff
@@ -692,6 +694,243 @@ public void HandleActionGetAndWieldItem(uint itemId, int wieldLocation)
+                                                                {43, 21082},
+                                                                {44, 21083},
+                                                                {45, 21084},
+                                                                {46, 21085},
+                                                                {47, 21086},
+                                                                {48, 21087},
+                                                                {49, 21088},
+                                                                {50, 21089},
+                                                                {51, 21055},
+                                                                {52, 21059},
+                                                                {53, 20981},
+                                                                {54, 21052},
+                                                                {55, 20991},
+                                                                {56, 0}, //metal
+                                                                {57, 21042},
+                                                                {58, 20982},
+                                                                {59, 21045},
+                                                                {60, 20984},
+                                                                {61, 20986},
+                                                                {62, 21068},
+                                                                {63, 21077},
+                                                                {64, 20993},
+                                                                {65, 0}, //stone
+                                                                {66, 20980},
+                                                                {67, 20985},
+                                                                {68, 21061},
+                                                                {69, 21063},
+                                                                {70, 21073},
+                                                                {71, 21075},
+                                                                {72, 0}, //wood
+                                                                {73, 21047},
+                                                                {74, 20988},
+                                                                {75, 20989},
+                                                                {76, 20990},
+                                                                {77, 21080}
+                                                            };
+            //CreatureSkill skill = GetCreatureSkill(Skill.Salvaging);
+            double salvageSkill = (double)GetCreatureSkill(Skill.Salvaging).Current;
```

Comment:
The skill check used any one of the different tinkering skills or the salvaging skill itself, whichever was higher, for any of the skill checks used.